monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -498,11 +502,32 @@ def train_single_model(self, config: Any, fold: int, gpu_id: int = 0, **kwargs:
                     - disable_checkpointing: True to disable checkpointing. Ideal for testing things out and you
                         don't want to flood your hard drive with checkpoints. Default: False.
         """
-        os.environ["CUDA_VISIBLE_DEVICES"] = f"{gpu_id}"
+        if isinstance(gpu_id, str):
+            gpu_id = tuple(map(int, gpu_id.replace('"', "").split(",")))
```

Comment:
I think it may be safer to make `gpu_id` a list of int numbers, in case user provides some weird input with comma.
```
gpu_id = [int(x) for x in gpu_id.split(",")]
```

Reply:
Inputting a list through Google Fire in the command line can be challenging for users.

Reply:
Hi @dongyang0122 , here the input `gpu_id` is a string that's separated by comma `,`. This line converts it to a tuple.

Reply:
I have resolved the tuple input issue stemming from Google Fire. However, we need to reach a consensus on whether the input type should be an integer or a tuple.

Reply:
In my understanding
integer is for single GPU
tuple is for single GPU or multiple GPU

Did I miss something that is important?

Reply:
I am unable to recall the context surrounding that message. I have introduced support for tuple, list and integer types.